covid-nlp/main.ipynb

```diff
@@ -0,0 +1,2384 @@
+{
```

Comment:
<p>Add a "bringing it all together",</p><p>Where you show all the rgxlog and the python code which is not a "generic ie" function together,</p><p>to show how small an implementation in our paradigm here is.</p><p>Also add metrics as we discussed (number of lines that are python,csv,rgxlog) etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>done</p>